infer_structcol/run_structcol.py

```diff
@@ -51,8 +118,8 @@ def calc_refl_trans(volume_fraction, radius, thickness, Sample, ntrajectories=60
     # Calculate the effective index of the sample
     sample_index = ri.n_eff(particle_index, matrix_index, volume_fraction)        
     
-    reflectance = []
-    transmittance = []
+    reflectance = np.zeros(len(wavelength))
+    transmittance = np.zeros(len(wavelength))
```

Comment:
Is wavelength a global variable here? It's not an argument of the function, so I'm guessing it must be? However, In calc_refl_trans the wavelength is taken from the Sample object instead of being called as a global variable. Is there a reason for doing it differently in these two functions?